eemeter/caltrack.py

```diff
@@ -1546,7 +1623,45 @@ def metered_savings_func(row):
         })
         results = results.join(counterfactual_usage_disaggregated)
 
-    return results.dropna().reindex(results.index)
+    error_bands = None
+
+    if frequency == 'daily' or frequency == 'billing':
+        num_parameters = model_results.totals_metrics.num_parameters
+
+        base_obs = model_results.totals_metrics.observed_length
+        post_obs = results['reporting_observed'].dropna().shape[0]
+
+        rmse_base_residuals = model_results.totals_metrics.rmse_adj
+        autocorr_resid = model_results.totals_metrics.autocorr_resid
+
+        base_avg = model_results.totals_metrics.observed_mean
+        post_avg = results['reporting_observed'].mean()
+        post_prediction_avg = results['counterfactual_usage'].mean()
+
+        base_var = model_results.totals_metrics.observed_variance
+
+        nprime = base_obs * (1 - autocorr_resid) / (1 + autocorr_resid)
+
+        total_base_energy = base_avg * base_obs
+
+        ols_total_agg_error, ols_model_agg_error, ols_noise_agg_error =_compute_ols_error(
+            t_stat, rmse_base_residuals, post_obs, base_obs, base_avg, post_avg, base_var,
+            nprime
+        )
+
+        fsu_error_band = _compute_fsu_error(
+            t_stat, frequency, post_obs, total_base_energy, rmse_base_residuals, base_avg,
+            base_obs, nprime
+        )
+
+        error_bands = {
```

Comment:
Note to self: possibly make lowercase, change whitespace for underscores for consistency?